src/OpenLoco/src/Windows/Options.cpp

```diff
@@ -1867,7 +1867,7 @@ namespace OpenLoco::Ui::Windows::Options
         static void edgeScrollingMouseUp(Window* w)
         {
             auto& cfg = OpenLoco::Config::get().old;
-            cfg.edgeScrolling = ~cfg.edgeScrolling;
+            cfg.edgeScrolling = !cfg.edgeScrolling;
```

Comment:
This would fix the issue but looking at it I see why we are getting the issue and its because `static void setDefaultsLegacyConfig()` doesn't set all of the defaults to the values I would expect. `setDefaultsLegacyConfig` will need a bit of a checkup by a reverse engineer though to fix the root cause. In the mean time this PR will do. All that is left to do is add a changelog entry in CHANGELOG.md. You'll see the layout for our other release just copy the same style and use this PR number as the Fix number (#1772).

Reply:
@duncanspumpkin Sorry, I got lost on this thread. Do you want me to update the changelog.md as a separate pull request?